versions.json

```diff
@@ -11,12 +11,12 @@
     },
     "spdk": {
         "repo": "https://github.com/longhorn/spdk.git",
-        "commit": "1975133e993723bc9ec3bd46862f4e5efe3b3fd1",
+        "commit": "42b649ffc6dcbc1797f32db4b70716b9c602ee39",
```

Comment:
As discussed, we need to automatically update the commit of `spdk` and `go-spdk-helper` here for the default branch which is used for the current development when there are new commits pushed to the default branch of those repos.